--- ./src/motionblur/MotionBlurCamera.cpp.orig	2014-09-10 21:15:35.000000000 +0400
+++ ./src/motionblur/MotionBlurCamera.cpp	2014-09-10 21:16:20.000000000 +0400
@@ -58,7 +58,7 @@
     framebuffer.setData(ColorFormat::RGB, ColorType::UnsignedByte, size, nullptr, BufferUsage::DynamicDraw);
     delete texture;
 
-    Buffer::unbind(Buffer::Target::PixelPack);
+    //Buffer::unbind(Buffer::Target::PixelPack);
     for(Int i = 0; i != FrameCount; ++i)
         frames[i]->setImage(0, TextureFormat::RGB8, framebuffer);
 }
